components/org.wso2.carbon.identity.media.core/src/main/java/org/wso2/carbon/identity/media/core/file/FileBasedStorageSystemImpl.java

```diff
@@ -79,7 +78,7 @@
 
     @Override
     public String addMedia(List<InputStream> inputStreams, MediaMetadata mediaMetadata, String uuid,
-                           String tenantDomain) throws StorageSystemException {
+                           String tenantDomain) throws StorageSystemServerException {
```

Comment:
why was this changed? don't we need to send a client exception in uuid already exists?

Reply:
uuid is system generated. hence there is no check for uuid already exists condition.